src/Agent.Worker/Handlers/StepHost.cs

```diff
@@ -97,12 +98,15 @@ public string ResolvePathForStepHost(string path)
             // remove double quotes around the path
             path = path.Trim('\"');
 
+            // if running on OS X, just translate the path
+            if (PlatformUtil.RunningOnMacOS)
+            {
+                return Container.TranslateToContainerPath(path);
+            }
+
             // try to resolve path inside container if the request path is part of the mount volume
-#if OS_WINDOWS
-            if (Container.MountVolumes.Exists(x => path.StartsWith(x.SourceVolumePath, StringComparison.OrdinalIgnoreCase)))
-#else
-            if (Container.MountVolumes.Exists(x => path.StartsWith(x.SourceVolumePath)))
-#endif
+            StringComparison comparator = (PlatformUtil.RunningOnWindows) ? StringComparison.OrdinalIgnoreCase : StringComparison.OrdinalIgnoreCase;
```

Comment:
Your comparers are the same. I assume you meant for the linux one to be Ordinal.

Reply:
Thanks!